components/bigcommerce/bigcommerce.app.mjs

```diff
@@ -1,11 +1,90 @@
+import { axios } from "@pipedream/platform";
+import constants from "./common/constants.mjs";
 export default {
   type: "app",
   app: "bigcommerce",
-  propDefinitions: {},
+  propDefinitions: {
+    storeHash: {
+      type: "string",
+      label: "Store Hash",
+      description: "Your store hash.",
+    },
+    channel: {
+      type: "boolean",
+      label: "Using channel",
+      description: "Channel of the webhook",
+    },
+    channelId: {
+      type: "string",
+      label: "Channel Id",
+      description: "Id of the channel",
+    },
+    maxRequests: {
+      type: "integer",
+      min: 1,
+      max: 180,
+      label: "Max API Requests per Execution (advanced)",
+      description:
+        "The maximum number of API requests to make per execution (e.g., multiple requests are required to retrieve paginated results).",
+      optional: true,
+      default: 1,
+    },
```

Comment:
Is this prop needed?

Reply:
I found this in another action, it is to limit requests if the customer has a large number of products registered

Reply:
Sure, but it seems that it's not used anywhere